api/src/main/resources/workbench.yaml

```diff
@@ -2317,7 +2317,7 @@ definitions:
   WorkspaceActiveStatus:
     type: string
     description: Status of workspace
-    enum: [ACTIVE, DELETED]
+    enum: [ACTIVE, DELETED, INACTIVE]
```

Comment:
We should document somewhere (probably here) what INACTIVE means

Reply:
+1 to this. This would also be a reasonable place to clarify what the expectations are around the DELETED state (e.g. is the workspace still "alive" in Terra? Has the underlying project been deleted? Curious minds want to know... we have comments here https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java#L312, but it should probably be documented at the API level.)

Re: naming... do we expect the INACTIVE state to be used by other migrations / workspaces in the future? Do we expect these workspaces to remain in this status indefinitely?

If this will effectively be a one-off status for migrations like this, it might be reasonable to make a more specific-sounding status like PENDING_DELETION_POST_MIGRATION.

Reply:
Would workspaces that had tripped the exfiltration alarm and been unplugged also be `INACTIVE` here? Or would that be a new status?

Reply:
Probably a new status. It's cheap to add a new status and hard (at the moment) to figure out how a workspace ended up in a certain state.

I can give it the more specific name as @gjuggler  suggested.

Which should also take care of the need for a comment. I think the name is self explanatory.